commands/acl-cat.md

```diff
@@ -1,5 +1,5 @@
 The command shows the available ACL categories if called without arguments.
-If a category name is given, the command shows all the Redis commands in
+If a category name is given, the command shows all the Valkey commands in
```

Comment:
Consider avoiding the name in texts like this. (?)

```suggestion
If a category name is given, the command shows all the commands in
```

Reply:
I'd prefer Valkey for now

1) it is not wrong 
2) it helps us get to the finish line (of removing Redis)
3) we can always swap it out later with "server". none of these words are binding.